internal/client/consensus/grandpa/justification_test.go

```diff
@@ -81,6 +81,17 @@ func TestJustificationEncoding(t *testing.T) {
 	require.Equal(t, expected, justification.Justification)
 }
 
+func TestDecodeJustificationFromWestendBlock2323464(t *testing.T) {
```

Comment:
Is this the test that reproduces the issue? If yes, why is it testing block 2323464 instead of 2318338?

Reply:
No, this one is meant to fix another issue that arose while I was reviewing the other one. I'll be adding tests for 2318338 once I finish addressing this [comment](https://github.com/ChainSafe/gossamer/pull/4336#discussion_r1863713207)